ansible-role-requirements.yml

```diff
@@ -14,10 +14,6 @@
   scm: git
   src: https://github.com/rsoprivatecloud/openstack-ops
   version: f87406880f67584b5fe562d2711734bbd63c0fdb
-- name: os_octavia
```

Comment:
[As discussed in slack](https://rackspace.slack.com/archives/C0EUKPSFR/p1498049639362131) we should probably leave the octavia bits.